basic-auth/views/error.hbs

```diff
@@ -0,0 +1,3 @@
+<h1>{{message}}</h1>
+<h2>{{error.status}}</h2>
+<pre>{{error.stack}}</pre>
```

Comment:
Let's try to not show the message to the user. Put a simple message here with 'Server error'